.github/workflows/smoke-tests.yaml

```diff
@@ -0,0 +1,49 @@
+name: consent-smoke-tests
+
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  smoke-tests:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 17
+      - uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Bee Create
+        uses: broadinstitute/workflow-dispatch@v3
+        with:
+          workflow: bee-create
+          repo: broadinstitute/terra-github-workflows
+          ref: refs/heads/main
+          token: ${{ secrets.BROADBOT_TOKEN}}
+          inputs: '{ "bee-name": "${{ github.event.repository.name }}-${{ github.run_id }}-dev", "bee-template-name": "duos", "version-template": "dev" }'
```

Comment:
I wonder if there's a way to create a named binding for an expression so you don't need to repeat it everywhere, like `"${{ github.event.repository.name }}-${{ github.run_id }}-dev"`. You could do it using a step or job output that just created the bee name, not sure how much extra code that would be though.

Reply:
There definitely is ... just haven't gotten there yet. Essentially you set it as the output and it's then a reference-able var